docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-overview.md

```diff
@@ -21,7 +21,8 @@ The following best practices are recommended for production clusters.
 | [Provider and region](#provider-and-region) | Deploy your cluster in a virtual private cloud (VPC), with the same provider and in the same region as your application VPC. YugabyteDB Managed supports AWS and GCP.<br>Multi-region clusters must be deployed in VPCs. You need to create the VPCs before you deploy the cluster. Refer to [VPC network](../cloud-vpcs/). |
 | [Fault tolerance](#fault-tolerance) | Region or Availability zone (AZ) level - minimum of three nodes across multiple regions or AZs, with a replication factor of 3. |
 | [Sizing](#sizing) | For most production applications, at least 3 nodes with 4 to 8 vCPUs per node.<br>Clusters support 10 simultaneous connections per vCPU. For example, a 3-node cluster with 4 vCPUs per node can support 10 x 3 x 4 = 120 connections.<br>When scaling your cluster, for best results increase node size up to 16 vCPUs before adding more nodes. For example, for a 3-node cluster with 4 vCPUs per node, scale up to 8 or 16 vCPUs before adding a fourth node. |
-| [YugabyteDB version](#yugabytedb-version) | Use the **Stable** release track.<!--<br>Use a [staging cluster](#staging-clusters) to test upgrades before upgrading your production cluster.--> |
+| [YugabyteDB version](#yugabytedb-version) | Use the **Stable** release track. |
+| [Staging cluster](#staging-cluster) | Use a staging cluster to test application compatibility with database updates before upgrading your production cluster. |
```

Comment:
@juanman2   Customers can chose to pin the staging cluster to a newer release or the prod cluster to an older release?

@ddhodge   do we have documentation on how to do the "pin to an older release" to link that documentation here?

Reply:
@tylarb Not sure what you are referring to. You can't pin to an older release. Far as I know all you can do is upgrade your staging cluster to test, before you upgrade your production cluster.